pkgs/tools/games/slipstream/default.nix

```
@@ -0,0 +1,43 @@
+{ lib, fetchFromGitHub, stdenv, makeWrapper, buildMaven, maven, jdk8 }:
+let
+  jdk = jdk8;
+  mavenWithJdk = maven.override { inherit jdk; };
+in
+mavenWithJdk.buildMavenPackage rec {
+  pname = "slipstream";
+  version = "1.9.1";
+
+  src = fetchFromGitHub {
+    owner = "Vhati";
+    repo = "Slipstream-Mod-Manager";
+    rev = "v${version}";
+    hash = "sha256-F+o94Oh9qxVdfgwdmyOv+WZl1BjQuzhQWaVrAgScgIU=";
+  };
+
+  mvnHash = "sha256-oDtUitsfZPiDtyfzzw1yMNBCKyP6rHczKZT/SPPJYGE=";
+
+  nativeBuildInputs = [ mavenWithJdk makeWrapper ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/java
+    install -Dm644 target/ftl-mod-manager-${version}.jar $out/share/java
+    install -Dm644 target/modman.jar $out/share/java
+
+    makeWrapper ${wrapper} $out/bin/${pname} \
+      --suffix PATH : ${lib.makeBinPath [ jdk ]} \
+      --set jar_file "$out/share/java/modman.jar"
```

This would remove the need for a `wrapper.sh` file:
```suggestion
    makeWrapper ${wrapper} $out/bin/${pname} \
      --suffix PATH : ${lib.makeBinPath [ jdk ]} \
      --run 'cd $XDG_DATA_HOME/slipstream'
      --run 'mkdir -p $_base/{mods,backup}'
      --run 'java -jar $out/share/java/modman.jar'
```

---

Implemented this & fallback logic in case `XDG_DATA_DIR` is undefined.